apps/shopify/src/dataTransformer.js

```diff
@@ -9,12 +9,14 @@ import flatten from 'lodash/flatten';
 export const dataTransformer = product => {
   const image = get(product, ['image', 'src'], '');
   const sku = get(product, ['sku'], '');
+  const variantSKU = get(product, ['variantSKU'], '');
 
   return {
     id: product.id,
     image,
     name: product.title,
-    sku
+    readableSKU: variantSKU,
```

Comment:
By calling it `readableSKU`, we limit this to be an sku. Why not just call it something like `subtitle`? That way people can put everything they want in it e.g. color, size etc.

Reply:
good idea